modules/nw-udn-cr.adoc

```diff
@@ -0,0 +1,96 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-udn-cr_{context}"]
+= Creating a UserDefinedNetwork custom resource
+
+The following procedure sets up a user-defined network that is namespace scoped.
+
+.Prerequisites
+* You must
+.Procedure
+
+[NOTE]
+====
+If any cluster default networked pods exist before the user-defined network is created, any further pods created in this namespace will return an error message: `What_is_this`.
+====
+
+. Create a request for a user-defined network:
+
+.. Create a YAML file, such as `my-udn-request.yaml`, to define your request with content like the following example:
++
+[source, yaml]
+----
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: udn-1 # <1>
+  namespace: default
+spec:
+  topology: Layer2 # <2>
+  layer2:
```

Comment:
This is also a parameter too, that means, in layer3 crd, this will be 'Layer3'

Reply:
I added it in some text to indicate this. Could you please re-check @huiran0826